packages/strapi-hook-bookshelf/lib/utils/connectivity.js

```diff
@@ -76,6 +76,7 @@ module.exports = (scope, success, error) => {
     });
   })
     .catch((err) => {
+      console.log(err)
```

Comment:
There is a `debug` option to check if we need to display the full log or not. We should remove this one, right?

Reply:
Updated